milestone2.ts

```diff
@@ -93,23 +101,39 @@ let findAverage: GetNumber; // Define lambda function here
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+
+let findIndexAns: GetNumber = (data) => {
+  for (let i=0; i<data.length; i++) {
+    if (data[i].city === "SF") {
+      return i;
+    }
+  }
+  return -1;
+}
 
 //console.log(findIndexAns(userData));
 
 // Question 3: Filtering data
 /* Use data handling function(s) to find all of the people from California (CA) over an age threshold n
  */
 type GetVolunteers = (data: Volunteer[], minAge: number) => Volunteer[];
-let findCAOverN: GetVolunteers; // Code here
+let findCAOverN: GetVolunteers = (data, minAge) => {
+  let cali = data.filter(person => person.state === "CA");
+  let ageThresh = cali.filter(p => p.age > minAge);
```

Comment:
You can define two conditions inside the same filter by using logical operators. so if you want to define two conditions that need to met you could use AND (&&). For example, filter(condition1 && condition2)